gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/DefaultTokenStateService.java

```diff
@@ -16,15 +16,19 @@
  */
 package org.apache.knox.gateway.services.token.impl;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
I recall comments from Kevin discouraging the use of this, but I don't recall the rationale.

Reply:
It was more around removing the commons-lang3 dependency if it could be avoided. I only bring it up if we don't need commons-lang3 everywhere. commons-lang3 wasn't added in this PR.